test/cli/console_test.rb

```diff
@@ -61,7 +61,7 @@ def create
         write.puts "find create!"
         read.gets
         output = read_for(read, pattern: /^> $/)
-        assert_match %r/#{Regexp.escape "User.create!(params[:user])"}/, output
+        assert_match /#{Regexp.escape "User.create!(params[:user])"}/, output
```

Comment:
This change is for the test of Actions. We should fix it as follows:

```suggestion
        assert_match %r/#{Regexp.escape "User.create!(params[:user])"}/, output
```